File: dart-programming/examples/intro/strings.dart
main() {
  var str = "Hello"  "World";
  print(str);        // HelloWorld
  str += " of Dart";
  print(str);        // HelloWorld of Dart
  
  print(str.length); // 18 
  
  print(str[0]);    // H
  
  var here = """
    Multi-line string
    instead of Here documents.
  """;
  print(here);
  
  
  var regular = 'regular strings \d \n\w';
  var raw = r'raw strings \d \n\w for regex';
  print(regular); // regular strings d 
                  // w
  print(raw);     // raw strings \d \n\w for regex
}

File: golang/examples/strings/strings.go
package main

import (
	"fmt"
)

func main() {
	text := "Hello World!"
	fmt.Println(text)
	fmt.Println(text[1])
	fmt.Printf("%c\n", text[1])

	text = text[:5] + "-" + text[6:]
	fmt.Println(text)
}
